src/main/views/template.njk

```diff
@@ -89,7 +89,7 @@
     [
       {
         text: navLinks.link1,
-        href: '/view-option'
+        href: '/'
```

Comment:
As discussed I reckon this will also need doing on the sign in pages

Reply:
done now :)